app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -356,6 +360,16 @@ public void onCreate(Bundle savedInstanceState) {
         floatingBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+<<<<<<< HEAD
```

Comment:
Yeah, I didn't see them before I pushed (after I thought I had successfully merged the files). I was in a rush and haven't updated it yet.